Kinder/LikedItems.xaml

```diff
@@ -7,6 +7,27 @@
         mc:Ignorable="d"
         Title="LikedItems" Height="450" Width="800">
     <Grid>
-        
+
+        <DataGrid x:Name="ItemsTable" HorizontalAlignment="Center" Margin="10,10,0,0" VerticalAlignment="Top" Width="700" Height="300" Grid.ColumnSpan="2">
+
+            <DataGrid.Resources>
+                <Style TargetType="{x:Type DataGridColumnHeader}">
+                    <Setter Property="Background" Value="Gray"></Setter>
+                    <Setter Property="FontWeight" Value="bold"></Setter>
+                    <Setter Property="BorderThickness" Value="0, 0, 1, 2"></Setter>
+                    <Setter Property="BorderBrush" Value="Black"></Setter>
+                </Style>
+            </DataGrid.Resources>
+
+            <DataGrid.Columns>
+                <DataGridTextColumn Header="item id" Binding="{Binding ItemID}" Width="50"></DataGridTextColumn>
+                <DataGridTextColumn Header="item name" Binding="{Binding ItemName}" Width="100"></DataGridTextColumn>
+                <DataGridTextColumn Header="Description" Binding="{Binding ItemDesc}" Width="*"></DataGridTextColumn>
+                <DataGridTextColumn Header="username" Binding="{Binding Username}" Width="100"></DataGridTextColumn>
+                <DataGridTextColumn Header="user email" Binding="{Binding Email}" Width="100"></DataGridTextColumn>
+             </DataGrid.Columns>
+
+        </DataGrid>
```

Comment:
Window formatting is done differently, can't properly comment without merging as there's no image of the window attached, but it's most likely fine.